view/adminhtml/layout/algolia_algoliasearch_queue_log.xml

```diff
@@ -3,6 +3,7 @@
       xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceContainer name="content">
+            <block class="Magento\Backend\Block\Template" name="analytics.overview.form" template="Algolia_AlgoliaSearch::queue/log/helper.phtml" />
```

Comment:
Not a big deal but I guess having a name like "algolia.indexing.log.helper" would be more relevant.